matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/database/query/UnsignedContent.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2020 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.matrix.android.internal.database.query
+
+internal object UnsignedContent {
+    internal const val REDACTED_TYPE = """{*"redacted_because":*}"""
```

Comment:
Maybe change FilterContent by TimelineEventFilter and add the RedactedType instead of creating a new object? (as you want)

Reply:
I've improved that